examples/fc_example/fc_example.ino

```diff
@@ -401,6 +401,12 @@ struct myTheme: public colorMapMono {
   // FC_MAP_COLOR(shift, nocolor);
 };
 
+#ifndef FC_HAVE_COLOR_FUNCTIONS
```

Comment:
I just added these constexpr for testing purposes. They can be removed as soon as the example sketch is fixed in a way that it does not exceed the PROGMEM limit.